src/TourManager/TourManager.Service.Model/Activity.cs

```diff
@@ -12,7 +12,7 @@ public class Activity
 		public string Phone { get; set; }
 		public string Email { get; set; }
 		public string Website { get; set; }
-		public ActivityOption ActivityType { get; set; }
+		public ActivityType ActivityType { get; set; }
```

Comment:
we can already rename also property to just Type, will be public ActivityType Type { get; set; }

Reply:
same applies to other properties, PropertyType.....etc

Reply:
done

Reply:
I think on the end we will get all this data from appropriate service and we will delete it from our side, to be more flexible. But for now its ok.